hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+Warn in :func:`~hypothesis.strategies.from_type` if the inferred strategy
+has no variation (always returning default instances). Also ensures
+variation in numpy data types by inferring h.extras.numpy.from_dtype().
```

Comment:
Style issue: I'd prefer to make this a cross-reference (like `from_type()` above), and say _what_ we infer it for.